addon/-private/helpers.js

```diff
@@ -1,9 +1,11 @@
-import Ember from 'ember';
+export { assign } from '@ember/polyfills';
```

Comment:
I think this introduced a dependency on a particular version of ember.

In our apps (currently using ember `2.16.2`) I got an error when I upgraded to `ember-cli-page-object` `1.15.0-beta.2` (so that I could use the goodness from this PR):

> Could not find module `@ember/polyfills` imported from `ember-cli-page-object/-private/helpers`

I can't remember is `assign` from Ember does anything special but I tried replacing this line with:

```js
export const assign = Object.assign;
```

locally and it *seems* to be working...

Reply:
can you reproduce that in a fresh `ember new` app? seems strange to me because the `ember-cli-babel` dependency from the addon should take care of transpiling that correctly 🤔

Reply:
Thanks for the hint :)

I tried to reproduce it in a fresh `ember new` app based on the same version of `ember-cli` and... I can't! See https://github.com/vitch/test-page-object/tree/test-page-object

I'm going to dig further to try and find out what we are doing which is causing problems and I'll let you know if I manage to reproduce it...